dbos/_sys_db.py

```diff
@@ -692,6 +700,45 @@ def get_workflows(self, input: GetWorkflowsInput) -> GetWorkflowsOutput:
 
         return GetWorkflowsOutput(workflow_uuids)
 
+    def get_queued_workflows(
+        self, input: GetQueuedWorkflowsInput
+    ) -> GetWorkflowsOutput:
+
+        query = sa.select(SystemSchema.workflow_queue.c.workflow_uuid).join(
+            SystemSchema.workflow_status,
+            SystemSchema.workflow_queue.c.workflow_uuid
+            == SystemSchema.workflow_status.c.workflow_uuid,
+        )
+
+        if input.get("queue_name"):
+            query = query.where(
+                SystemSchema.workflow_queue.c.queue_name == input["queue_name"]
+            )
+
+        if input.get("status"):
+            query = query.where(
+                SystemSchema.workflow_status.c.status == input["status"]
+            )
+        if "start_time" in input and input["start_time"] is not None:
```

Comment:
why can't this be `if input.get("start_time")`, as above?

Reply:
`mypy` isn't happy